src/mscorlib/src/System/Reflection/RuntimeConstructorInfo.cs

```diff
@@ -462,18 +461,5 @@ public override Object Invoke(BindingFlags invokeAttr, Binder binder, Object[] p
             return RuntimeMethodHandle.InvokeMethod(null, null, sig, true);
         }
         #endregion
-
-        #region ISerializable Implementation
-        public void GetObjectData(SerializationInfo info, StreamingContext context)
-        {
-            throw new PlatformNotSupportedException();
```

Comment:
Is there any concern that calling GetObjectData on Runtime*Infos will throw NotImplemented, which is different than every other type we don't support serializing?